R/tax_range_strat.R

```diff
@@ -184,6 +197,10 @@ tax_range_strat <- function(occdf, name = "genus", level = "bed",
     ranges <- ranges[order(ranges$min_bin), ]
   }
 
+  if (!is.null(group)) {
+    ranges <- ranges[order(ranges$group), ]
```

Comment:
Same comment as with `tax_range_time()`

Currently, we're ordering by `group` after ordering `name` by how the user specified. I think when group is supplied, we should be ordering using `by` for `name` and `group` (presumably `group` first), e.g. `group` are all organised by FAD, and then name within `group` are organised by FAD too.

Reply:
Yes, precisely